src/components/list-address/AddressList.jsx

```diff
@@ -0,0 +1,39 @@
+import "./index.scss"
+
+const AddressList = ({ addresses, addAddress, removeAddress}) => {
+    const handleAddress = () => {
+        const address = prompt('Ingrese una nueva direccion ')
+        if (address){
+            addAddress(address)
+        }
+    }
+
+    const handleRemoveAddress = (index) => {
+        if(window.confirm('¿Estas seguro de eliminar esta dirección?')){
+            removeAddress(index)
+        }
```

Comment:
I consider a modal component would be better.

Reply:
Rodrigos's component